packages/react-native-gesture-handler/src/__tests__/api_v3.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { usePanGesture } from '../v3/hooks/gestures';
```

Comment:
Will this run on our GH Actions?

Reply:
I think we can set it up and work on the tests in the follow-ups.

Reply:
Why not just run `yarn test`?

Reply:
`yarn test` will run all of them, don't we want only tests for V3?